lua/neo-tree/sources/common/commands.lua

```diff
@@ -481,6 +481,34 @@ M.copy = function(state, callback)
   fs_actions.copy_node(node.path, nil, callback, using_root_directory)
 end
 
+---Copies a node relative path to clipboard.
+---@param state table The state of the source
+M.copy_path = function(state)
+  local node = state.tree:get_node()
+  if node.type == "message" then
+    return
+  end
+  local pwdpath = state.path
+  local content = node.path
```

Comment:
Since this is a common command, there's no guarantee that either of these exist for all sources.
```suggestion
  local pwdpath = state.path or ""
  local content = node.path or node:get_id()
```